docs/_admin/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Project Administration
+order: -1
+---
+
+The web-platform-tests project is more than just tests! This section documents
+all the information necessary to administer the infrastructure which makes the
+project possible.
+
+## Secrets
+
+Some aspects of the infrastructure are only accessible to administrators. These
+include access credentials for third-party services (e.g. continuous
+integration providers), data used to prove identity (e.g. SSL certificates),
+and deployment information for the project's running systems (e.g.
+[w3c-test.org](https://w3c-test.org)).
+
+Project                   | Secret                   | Owners
+--------------------------|--------------------------|-------------
+[results-collection]      | root SSH keys            | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[results-collection]      | Password for app secrets | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[pull-requests-dashboard] | root SSH keys            | ???
+[pull-requests-dashboard] | Password for app secrets | boaz@bocoup.com, geoffers@gmail.com, jgraham@hoppipolla.co.uk, lukebjerring@google.com, mike@w3.org
+[web-platform-tests]      | TravisCI authorization   | ???
```

Comment:
Travis CI authorisation for what?

Reply:
I'm not sure, exactly. [The project's `.travis.yml` file currently includes three encrypted values.](https://github.com/w3c/web-platform-tests/blob/6f03878da8255e23699c5d45e98dd123ef88383a/.travis.yml) Two seem to authorize uploading of generated test manifest files. The third is used by the "stability checker" job; I don't know what pridileged action that needs to take.

Reply:
The test manifest stuff just runs off of @jgraham's account; I think they're just personal access tokens so could be replaced by anyone's?

Don't know what the stability checker job has the secure env variables for; maybe the PR dashboard?

Reply:
Good question. I did some digging, and it looks like they're no longer necessary: https://github.com/w3c/web-platform-tests/pull/10937 . That, together with the fact that manifest upload credentials are synonymous with organization membership, means that this entry can be removed. Thanks!